fe/fe-core/src/main/java/com/starrocks/analysis/CreateFunctionStmt.java

```diff
@@ -102,7 +131,11 @@ public void analyze(Analyzer analyzer) throws UserException {
         if (isAggregate) {
             analyzeUda();
         } else {
-            analyzeUdf();
+            if (isStarrocksJar) {
```

Comment:
Why all name is `starrocksJar`, not `jar`?

Reply:
I was thinking maybe there are other format in JAR, like HIVE/Spark/Presto etc.